package.json

```diff
@@ -1,66 +1,79 @@
 {
   "private": true,
-  "license": "MIT",
+  "engines": {
+    "node": ">=6",
+    "npm": ">=3"
+  },
   "scripts": {
-    "install": "run-s -s install:*",
-    "install:mkdirp-bin": "lerna exec -- mkdirp ./node_modules/.bin/",
-    "install:bootstrap": "lerna bootstrap",
-    "install:symlink-bin": "lerna exec -- node ../../scripts/symlink-bin.js",
-    "clean": "lerna exec -- rm -rf lib/* && lerna clean --yes",
-    "lint": "lerna exec -- npm run lint -s",
-    "build": "lerna exec -- npm run build -s",
-    "test": "lerna exec -- npm run test -s",
-    "coverage": "lerna exec -- npm run coverage -s",
-    "codecov": "lerna exec -- npm run codecov -s",
-    "commit": "git-cz || true",
-    "check-updates": "run-p -s check-updates:*",
-    "check-updates:root": "ncu --error-level 2",
-    "check-updates:packages": "lerna exec -- ncu --error-level 2",
-    "upgrade": "run-p -s check-updates:*",
-    "upgrade:root": "ncu --loglevel minimal --upgrade",
-    "upgrade:packages": "lerna exec -- ncu --loglevel minimal --upgrade",
+    "postinstall": "check-node-version --package && lerna bootstrap",
+    "check-updates": "./scripts/check-updates",
+    "clean": "rm -rf coverage && ./scripts/run-task clean",
+    "build": "./scripts/run-task build",
```

Comment:
This won't work the way we want because we're not just looping.
The `./scripts/run-task` has specific logic (see https://github.com/commercetools/nodejs/pull/2/commits/e6310918887186a6a2e7a4ed0c1ee2d37f969f33)